spec/features/audit/report/csv_export_spec.rb

```diff
@@ -13,99 +7,140 @@ def content_disposition
+                      allocated_to: user)
+    create(:link,
+           source_content_id: example1.content_id,
+           target_content_id: hmrc.content_id,
+           link_type: "primary_publishing_organisation")
+    create(:audit, content_item: example1)
+  end
 
-      select "HMRC", from: "Organisations"
-      click_on "Apply filters"
+  def then_i_receive_only_hmrc_related_content_items_in_the_report
+    expect(@audit_report).to have_content("Title,URL")
+    expect(@audit_report).to have_content("Example 1,https://gov.uk/example1")
+    expect(@audit_report).to have_no_content("Example 2,https://gov.uk/example2")
+  end
 
-      click_link "Export filtered audit to CSV"
-      expect(page).to have_content("Example 1,https://gov.uk/example1")
-      expect(page).to have_no_content("Example 2,https://gov.uk/example2")
-    end
+  def then_i_see_the_two_unassigned_content_items
+    expect(@audit_content_page).to have_no_content("Example 1")
+    expect(@audit_content_page).to have_content("Example 2")
+    expect(@audit_content_page).to have_content("Example 3")
+  end
 
-    scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
-      visit audits_path
-      select "Anyone", from: "allocated_to"
+  def when_i_navigate_to_audits_report_page
+    @audit_content_page.audits_progress_tab.click
+  end
 
-      choose "Audited"
+  def then_i_do_not_see_the_two_unassigned_content_items
+    expect(@audit_report_page).to be_displayed
+    expect(@audit_report_page).to have_no_content('2 Content items')
+  end
 
-      click_on "Apply filters"
-      expect(page).to have_content("Example 1")
-      expect(page).to have_no_content("Example 2")
+  def and_i_see_the_one_content_item_assigned_to_me
+    expect(@audit_report_page).to have_content('1 Content items')
+  end
 
-      click_link "Audit progress"
+  def when_i_export_an_audit_report
+    @audit_report = ContentAuditTool.new.audit_report_page
+    @audit_report.load(content_id: Content::Item.last.content_id)
+    @audit_report.export_to_csv.click
+  end
 
-      click_link "Export filtered audit to CSV"
-      expect(content_disposition).to include(
-        'filename="Transformation_audit_report_CSV_download.csv"',
-      )
-      expect(page).to have_content("Example 1")
-      expect(page).to have_no_content("Example 2")
-    end
+  def then_i_receive_the_report_in_csv_format
+    expect(content_type).to eq("text/csv")
+    expect(content_disposition).to start_with("attachment")
+    expect(content_disposition).to include(
+      'filename="Transformation_audit_report_CSV_download.csv"',
+    )
+    expect(@audit_report).to have_content("Title,URL")
+    expect(@audit_report).to have_content("Example 1,https://gov.uk/example1")
+  end
+
+  def and_filter_is_applied_to_show_only_hmrc_content_items
+    @audit_report = ContentAuditTool.new.audit_report_page
+    @audit_report.load(content_id: Content::Item.last.content_id)
```

Comment:
What does this line do? Why are we loading a particular content item for the report?

Reply:
√